ec2uploadimg

```diff
@@ -685,10 +685,7 @@ def get_ssh_user(args, config, region, logger):
 # ----------------------------------------------------------------------------
 def get_vpc_subnet_id(args, config, region, setup,  logger):
     """Function to get the vpn_subnet parameter"""
-    vpc_subnet_id = args.vpcSubnetId
-
-    if not vpc_subnet_id and not args.accountName:
-        vpc_subnet_id = setup.create_vpc_subnet()
+    vpc_subnet_id = args.vpcSubnetId or setup.create_vpc_subnet()
```

Comment:
This is not correct. With this new code a new subnet is created if if an account name is specified. We do not give the code the chance to lookup if there is a subnet specified in the config that should be used. The subnet should **only** be created if we cannot find a setting.

Reply:
The current behaviour is when we try to find in the config and it's not found, the program raises an exeption. The correct behaviour is try to lookup the subnet id in the config file, if it's not found, try to create a new one. If there are no way to create a subnet should we raise an exeption? The error message should be different now, right?